app/controllers/api/v0/competitions_controller.rb

```diff
@@ -37,6 +37,14 @@ def competition_index
              include: serial_includes
   end
 
+  def competition_count
+    start_date = params.require(:startDate)
+    end_date = params.require(:endDate)
+
+    count = Competition.where("start_date >= ? AND end_date <= ?", start_date, end_date).count
+    render json: { count: count }
+  end
```

Comment:
This is a highly specific endpoint for internal uses, I'd rather not have that as part of the public API.

Reply:
But you forgot to remove the code here :wink: